development/auto-changelog.js

```diff
@@ -75,12 +75,12 @@ async function main() {
   const isReleaseCandidate = mostRecentVersion !== version;
   const versionHeader = `## [${version}]`;
   const escapedVersionHeader = escapeRegExp(versionHeader);
-  const currentDevelopBranchHeader = '## Current Develop Branch';
+  const currentDevelopBranchHeader = '## [Unreleased]';
```

Comment:
This may be a preexisting issue, but I'm confused by the error thrown in side the `if` statement on L88 below, which will always have the message `Failed to find release header '## [Unreleased]'`, which should always (in terms of its natural language meaning) be false.

Reply:
That error is to protect against the changelog getting malformed. If it can't find the "Unreleased" header, it can't know where to place the changelog entries so it blows up.

Reply:
Some defensive programming. I'm all for it!